### ISSUE TITLE
Support for LUKS detached headers

### ISSUE THREAD

**Issue by PoisonFrog** [storaged-project/udisks#944]
Can support for LUKS detached headers be added? It is possible to encrypt a block device with LUKS and detach the header, it would be nice if we could point to a detached header when trying to unlock a device with udisks.

**Comment by tbzatek** [comment]
Yes, this would be a valid and useful use case. There is some support in `libblockdev` already, needs enhancements in `org.freedesktop.UDisks2.Encrypted`.
